lesson_9/homework_09.py

```diff
@@ -0,0 +1,32 @@
+class Rhombus:
+    def __init__(self, side_a: float, angle_a: float):
+        self.side_a = side_a
+        self.angle_a = angle_a
+    
+    # Control setting of attributes 
+    def __setattr__(self, name, value):
+        # Make rules for our attributes
+        if name == "side_a":
+            if value <= 0:
+                raise ValueError("Side must be more than 0.")
+        elif name == "angle_a":
+            if not (0 < value < 180):
+                raise ValueError("Angle must be between 0 and 180")
+            object.__setattr__(self, "angle_b", 180 - value)
+        
+        # Approve and set attributes
+        object.__setattr__(self, name, value)
+    
+    # Dont allow to set angle β only for reading
+    @property
+    def angle_b(self):
+        return self._angle_b
```

Comment:
```suggestion
    def angle_b(self):
        return self._angle_b
```
ну тоді  вже ж додайте пропері на всі атрибути (благо їх усього три)